content/the-ember-times-issue-172.md

```diff
@@ -5,6 +5,7 @@ authors:
   - chris-ng
   - amy-lam
   - abhilashlr
+  - jen-weber
```

Comment:
Mind adding your name at the end of the post too?

Reply:
@chrisrng done! Thank you!